vmdb/app/helpers/api_helper/renderer.rb

```diff
@@ -175,8 +175,8 @@ def collection_search(is_subcollection, type, klass)
         else
           klass.scoped
         end
-      res = res.where(sqlfilter_param)            if sqlfilter_param
-      res = res.reorder(sort_params)              if sort_params
+      res = res.where(sqlfilter_param)            if sqlfilter_param && res.respond_to?(:where)
+      res = res.reorder(sort_params)              if sort_params && res.respond_to?(:reorder)
```

Comment:
for those 2 lines same as other PR, didn't want more PR dependencies so they can be merged separately. Thanks.